src/applications/disability-benefits/686c-674/containers/ConfirmationPage.jsx

```diff
@@ -25,46 +26,62 @@ export class ConfirmationPage extends React.Component {
     const { response } = submission;
     const veteranFirstName = data?.veteranInformation?.fullName?.first || '';
     const veteranLastName = data?.veteranInformation?.fullName?.last || '';
+    const dateSubmitted = moment(response?.timestamp);
 
     return (
       <>
-        <div>
-          <button onClick={this.handlePrintClick} className="usa-button">
+        <div className="print-only">
+          <img
+            src="https://www.va.gov/img/design/logo/logo-black-and-white.png"
+            alt="VA logo"
+            width="300"
+          />
+          <h2 className="vads-u-margin-top--2">
+            Add or remove a dependent on your VA disability benefits
+          </h2>
+          <div className="vads-u-margin-bottom--2">
+            VA Form 21-686c (with 21P-527EZ and 21-674)
+          </div>
+        </div>
+        <div className="inset">
+          <h2
+            id="thank-you-message"
+            className="vads-u-font-size--h3 vads-u-font-family--serif vads-u-margin-top--1"
+          >
+            Thank you for submitting your application
+          </h2>
+          <dl>
+            <dt>
+              <strong>
+                Application for Declaration of Status of Dependents (Form
+                21-686c), and/or
+                <br />
+                Request for Approval of School Attendance (Form 21-674), and/or
+                <br />
+                Application for Veterans Pension (Form 21P-527EZ)
+              </strong>
+            </dt>
+            <dd>
+              {veteranFirstName || veteranLastName
+                ? `for ${veteranFirstName} ${veteranLastName}`
+                : ''}
+            </dd>
+            <dt>
+              <strong>Date submitted</strong>
+            </dt>
+            <dd>
+              {dateSubmitted.isValid()
+                ? dateSubmitted.format('MMMM D, YYYY')
```

Comment:
Are we changing the format on the date?

Reply:
It's recommended to spell out the dates and [only abbreviate the month when space is very limited](https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years)